src/styles/main.scss

```
@@ -0,0 +1,86 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+@import "variables";
+
+body {
+  padding: 20px;
+  margin: 0;
+}
+
+.clock {
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: $clock-height;
+  width: $clock-height;
+  margin: auto;
+
+  background-color: #e2c0c2;
+  border-radius: 50%;
+
+  &__number {
+    font-family: Arial, sans-serif;
+    font-size: 100px;
+    color: #28545e;
+  }
+
+  @each $side in right, left {
+    @include number($side, 20);
+  }
+
+  @each $side in top, bottom {
+    @include number($side, 5);
+  }
+
+  &__minute {
+    @include position;
+
+    width: $minute-width;
+    height: $minute-height;
+
+    border-radius: 5px;
+    background-color: #d15559;
+
+    transform-origin: bottom;
+    animation: clock 3600s steps(60,end) infinite;
```

```suggestion
    animation: clock 3600s steps(60, end) infinite;
```

No strong opinion, but I think minute should not have steps